app/src/main/java/com/example/notebook/domain/use_cases/GetNotesUseCase.kt

```diff
@@ -0,0 +1,15 @@
+package com.example.notebook.domain.use_cases
+
+import android.util.Log
+import androidx.lifecycle.LiveData
+import com.example.notebook.domain.models.Note
+import com.example.notebook.domain.repository.NoteRepository
+import javax.inject.Inject
+
+class GetNotesUseCase @Inject constructor (private val noteRepository: NoteRepository) {
+
+    fun execute(): LiveData<List<Note>> {
+        Log.d("Repository DATA", noteRepository.getAllNotes().value.toString())
+        return noteRepository.getAllNotes()
```

Comment:
Doesn't seem like production level debugging
and you could utilize a scope function of **also**:
`return repo.method().also { 
    log(it.yourData())
}`